rocAL/include/loaders/circular_buffer.h

```diff
@@ -31,12 +31,15 @@ THE SOFTWARE.
 #include "commons.h"
 #include "device_manager.h"
 #include "device_manager_hip.h"
-struct decoded_image_info {
-    std::vector<std::string> _image_names;
+struct DecodedDataInfo {
+    std::vector<std::string> _data_names;
     std::vector<uint32_t> _roi_width;
     std::vector<uint32_t> _roi_height;
     std::vector<uint32_t> _original_width;
     std::vector<uint32_t> _original_height;
+    std::vector<uint32_t> _original_audio_samples; //! Amplitude of an audio signal at a specific point in time
```

Comment:
This is very confusing. We are combining audio related stuff to DecodedDataInfo. Does the roi_width, _original_width applies to audio as well?

Reply:
We tried to use union for this, but it is not working as expected we are exploring on other options, if not we can add separate struct for audio info and replicate API calls in circular buffer for the same

Reply:
Let's revisit this after getting audio working on the tot